### PR TITLE
Close out improve login notifications

### PR DESCRIPTION
**Pull request by vbudhram**
This PR adds the outcomes to the improved login notifications feature card.

Ref: https://github.com/mozilla/fxa-features/issues/5

### CODE REVIEW

File: features/FxA-99-improved-login-notification/README.md [hunk 1]
```diff
@@ -195,3 +195,21 @@ for inclusion when constructing the localized email.
   For example can users report a location that
   seems wildly inaccurate?
 
+### Results
```
Comment (vbudhram) [inline features/FxA-99-improved-login-notification/README.md]: @rfk Wasn't sure if this should go in the `Outcomes` section. Currently, that section talks about high level tasks and requirements, not really the outcomes?

File: features/FxA-99-improved-login-notification/README.md [hunk 2]
```diff
@@ -176,15 +176,15 @@ for inclusion when constructing the localized email.
 
 ## High-level work breakdown
 
-* [ ] Confirm or add ability to measure click-through
+* [x] Confirm or add ability to measure click-through
       rate from these emails.
-* [ ] Select and integrate an IP geo lookup service.
-* [ ] Integrate geo lookup in fxa-auth-server and pass
+* [ ] ~~Select and integrate an IP geo lookup service.~~
```
Comment (rfk) [inline features/FxA-99-improved-login-notification/README.md]: What does `~~` signify here?

Comment (rfk) [inline features/FxA-99-improved-login-notification/README.md]: Like, we didn't do it in the end?

**Comment by rfk** [comment]
Thanks @vbudhram!  One question I have, is whether the dashboards you're reporting here accurately map back to the proposed key metric of: *"the number of users that click a link in the signin notification email, as a percentage of the number of such emails sent"*.

The sign-in confirmation success graph is close to this, but doesn't fully capture all click-throughs, e.g. it doesn't capture clicks on "reset your password".

**Comment by rfk** [comment]
Also, I think it's likely that we can't actually manage this part: *"This will demonstrate that the new information has changed the way users engage with the emails:"*.  Because we shipped the new notifications before we actually had the click-through metrics.  If that is the case, we should say so explicitly.

File: features/FxA-99-improved-login-notification/README.md (hunk 2 above)
Comment (vbudhram) [inline features/FxA-99-improved-login-notification/README.md]: I misinterpreted this requirement, we did complete this, will update.

**Comment by vbudhram** [comment]
@rfk Thanks for review, added some additional supporting data. What are your thoughts on the success of this feature? The graphs and charts lead me to believe it was neutral to positive, but maybe that is not case? Mind another r?

**Comment by rfk** [comment]
Thanks @vbudhram!  I agree that we can comfortably declare this feature successful even without a big change in user behaviour to point to, since we know from user feedback that they expect these details in the email.

**Pull request merged**